setup.py

```diff
@@ -24,7 +24,7 @@
         'setuptools_scm>=2.0.0,<3',
         'tenacity>=4.10.0,<5',
         'PyJWT==1.6.4',
-        'hca-metadata-api @ git+git://github.com/HumanCellAtlas/metadata-api@release/1.0b17',
+        'hca-metadata-api @ git+git://github.com/HumanCellAtlas/metadata-api@release/1.0b19',
```

Comment:
It seems like there should be a way to avoid duplication here. A quick google turned up https://www.reddit.com/r/Python/comments/3uzl2a/setuppy_requirementstxt_or_a_combination/

It looks like there's a way to read in `requirements.txt` in `setup.py` but other commenters claim that you don't need both.

Reply:
At this moment, the `*requirements.txt` are not used anywhere else but the Travis test setup, so all we need is the setup.py if we can figure out how to let Travis install from setup.py. 

(Ideally we'd replace all of the setup.py, requirements.txt, manfest.in files with a TOML file and a lock file (https://github.com/sdispater/poetry/blob/master/pyproject.toml) But since `poetry` is still in Beta, we might need to wait a bit until it's adopted more widely)

Reply:
I think the `requirements.txt` are useful to keep for local development